R/ard_attributes.R

```diff
@@ -82,6 +82,11 @@ ard_attributes.data.frame <- function(data,
       ),
       context = "attributes"
     ) |>
+    dplyr::mutate(
+      fmt_fn = list(as.character),
+      warning = list(NULL),
+      error = list(NULL)
+    ) |>
```

Comment:
This is great, thank you for adding! Can you move these to the `mutate()` that appears directly above?

Also for `fmt_fn`, your specification is perfect. But can we change it to the below? I was thinking about it more, and people may assign anything as an attribute so we can't assume it's a string. But we can assume the `label` is a string.

```r
fmt_fn = ifelse(.data$stat_name %in% "label", list(as.character), list(NULL))
```

Reply:
Moved the column creation to the `mutate()` above (not sure why I didn't just put them there in the first place) and used the above definition for `fmt_fn` - thank you!